Design/DesignDoc.md

```diff
@@ -0,0 +1,220 @@
+
+U6. As a user, I want to be able to view a list of all my expenditures sorted by date (newest - oldest)
+
+U6. As a user, I want to be able to get a specific day’s worth of expenses.
+
+U7. As a user, I want to be able to get a specific week’s worth of expenses.
+
+U8. As a user, I want to be able to get a specific month’s worth of expenses.
+
+U9. As a user, I want to be able to get a specific year’s worth of expenses.
+
+U10. As a user, I want to be able to view a graph or chart that depicts my expenses for a certain period in an informative manner.
+
+
+
+## 4. Project Scope
+
+_Clarify which parts of the problem you intend to solve. It helps reviewers know what questions to ask to make sure you are solving for what you say and stops discussions from getting sidetracked by aspects you do not intend to handle in your design._
+
+
+
+### 4.1. In Scope
+
+_Which parts of the problem defined in Sections 1 and 2 will you solve with this design? This should include the base functionality of your product. What pieces are required for your product to work?_
+
+* Adding, updating, and retrieving/viewing expenditure information
+* User specific data (multiple users allowed)
+* Retrieving expenditure information based on tags
+
+### 4.2. Out of Scope
+
+_Based on your problem description in Sections 1 and 2, are there any aspects you are not planning to solve? Do potential expansions or related problems occur to you that you want to explicitly say you are not worrying about now? Feel free to put anything here that you think your team can't accomplish in the unit, but would love to do with more time._
+
+* This API does not link with any card provider or bank provider so
+* This API does not facilitate payment. It is simply a means of retrieving data in a meaningful manner.
+
+
+
+
+# 5. Proposed Architecture Overview
+
+_Describe broadly how you are proposing to solve for the requirements you described in Section 2. This may include class diagram(s) showing what components you are planning to build. You should argue why this architecture (organization of components) is reasonable. That is, why it represents a good data flow and a good separation of concerns. Where applicable, argue why this architecture satisfies the stated requirements._
+
+This initial iteration will provide the minimum viable product (MVP) including adding, retrieving, and updating an employee contact information for the admin role.
+
+We will use API Gateway and Lambda to create 5 endpoints (GetExpense, Add/UpdateExpense, GetTaggedExpense, GetBudget, Add/UpdateBudget) that will handle the creation, updation, and retrieval of expense and budget information to satisfy our requirements.
+
+We will store expense and budget information in a table in DynamoDB.
+
+Amazon Elite Budget Service will also provide a web interface for users. A main page providing a variety of options to add, update and view expense and budget information.
+
+
+
+
+# 6. API
```

Comment:
This section looks like you're still working on it, so just 2 quick comments. 

1. Make sure you're consistent with employee vs expense in your terminology (maybe the employee is a carry over from your midstone?). 
2. Make sure that your endpoints will enable you to build the functionality that you've described in your use cases.